samples/bluetooth/periodic_adv_conn/src/main.c

```diff
@@ -0,0 +1,212 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/conn.h>
+#include <zephyr/bluetooth/gap.h>
+
+#define NUM_RSP_SLOTS	  5
+#define NUM_SUBEVENTS	  5
+#define PACKET_SIZE	  5
+#define SUBEVENT_INTERVAL 0x30
+
+static const struct bt_le_per_adv_param per_adv_params = {
+	.interval_min = 0xFF,
+	.interval_max = 0xFF,
+	.options = 0,
+	.num_subevents = NUM_SUBEVENTS,
+	.subevent_interval = SUBEVENT_INTERVAL,
+	.response_slot_delay = 0x5,
+	.response_slot_spacing = 0x50,
+	.num_response_slots = NUM_RSP_SLOTS,
+};
+
+static struct bt_le_per_adv_subevent_data_params subevent_data_params[NUM_SUBEVENTS];
+static struct net_buf_simple bufs[NUM_SUBEVENTS];
+static uint8_t backing_store[PACKET_SIZE][NUM_SUBEVENTS];
```

Comment:
Should this be static uint8_t backing_store[NUM_SUBEVENTS][PACKET_SIZE]; ?

Reply:
This should be fixed upstream. Will you post a PR, @hermabe?

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/57591